modules/aggregation-layers/src/grid-layer/grid-layer.js

```diff
@@ -63,6 +64,10 @@ export default class GridLayer extends CompositeLayer {
       // accessor for custom color or elevation calculation is specified
       return false;
     }
+    if (colorScaleType === 'quantile' || colorScaleType === 'ordinal') {
```

Comment:
We have section in grid-layer.md that lists all fallback cases, we should update with this case.